client/src/components/ResourceList/index.js

```diff
@@ -1,29 +1,101 @@
 import PropTypes from "prop-types";
+import { useEffect, useState } from "react";
 
+import { FormControls } from "../../components";
+import { TopicService, useService, ResourceService } from "../../services";
 import "./ResourceList.scss";
 
-export default function ResourceList({ publish, resources }) {
+export default function ResourceList({ publish, resources, pathname }) {
+	const [topics, setTopics] = useState([]);
+	const topicService = useService(TopicService);
+	const [selectedTopic, setSelectedTopic] = useState("");
+	const [filteredResources, setFilteredResources] = useState([]);
+	const resourceService = useService(ResourceService);
```

Comment:
💡 I'd have done:
```suggestion
	const [filteredResources, setFilteredResources] = useState([]);
	const [selectedTopic, setSelectedTopic] = useState("");
	const [topics, setTopics] = useState([]);
	const resourceService = useService(ResourceService);
	const topicService = useService(TopicService);
```

💡 the default for `filteredResources` might as well be `resources`

💡 maybe `undefined` would be a better default topic than `""`?

Reply:
@textbook done thanks (:+1